client/src/components/UIElements/Button.tsx

```diff
@@ -0,0 +1,19 @@
+import "./Button.scss";
+
+type buttonProps = {
+  onClick?: () => void;
+  children: string;
+  inverse?: boolean;
+  bold?: boolean;
+};
+
+const Button = (props: buttonProps) => {
+
+  return (
+    <button className={`button ${props.inverse && 'inverse'} ${props.bold && 'bold'}`} onClick={props.onClick}>
+      {props.children}
+    </button>
+  );
+};
```

Comment:
when creating a basic HTML element (here you are extending button with your own style) you want to extend the button.
Check this out:
https://stackoverflow.com/a/62568833